<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_state">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>X coordinate</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Z speed</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Z coordinate</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>X Step</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="ZAxisPosition">
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="minimum">
                 <double>-100000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>100000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>X speed</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QSlider" name="ZSpeed">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QSlider" name="YSpeed">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Y coordinate</string>
                </property>
               </widget>
              </item>
              <item row="4" column="2" colspan="4">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="YAxisPosition">
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="minimum">
                 <double>-100000.000000000000000</double>
                </property>
                <property name="value">
                 <double>0.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="XAxisPosition">
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="minimum">
                 <double>-100000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1000000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QSlider" name="XSpeed">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Y speed</string>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QDoubleSpinBox" name="XStep">
                <property name="decimals">
                 <number>0</number>
                </property>
                <property name="minimum">
                 <double>-10000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QDoubleSpinBox" name="YStep">
                <property name="decimals">
                 <number>0</number>
                </property>
                <property name="minimum">
                 <double>-10000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="4">
               <widget class="QDoubleSpinBox" name="ZStep">
                <property name="decimals">
                 <number>0</number>
                </property>
                <property name="minimum">
                 <double>-10000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Y Step</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Z Step</string>
                </property>
               </widget>
              </item>
              <item row="1" column="5">
               <widget class="QPushButton" name="MoveYPlus">
                <property name="text">
                 <string>&gt;</string>
                </property>
               </widget>
              </item>
              <item row="2" column="5">
               <widget class="QPushButton" name="MoveZPlus">
                <property name="text">
                 <string>&gt;</string>
                </property>
               </widget>
              </item>
              <item row="6" column="2" colspan="4">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0" colspan="6">
               <widget class="QPushButton" name="SetCoordinate">
                <property name="text">
                 <string>Set Coordinate</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0" colspan="6">
               <widget class="QPushButton" name="SetTranslationStageSpeed">
                <property name="text">
                 <string>Set Translation Stage Speed</string>
                </property>
               </widget>
              </item>
              <item row="5" column="2" colspan="4">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QPushButton" name="MoveXMinus">
                <property name="text">
                 <string>&lt;</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QPushButton" name="MoveYMinus">
                <property name="text">
                 <string>&lt;</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QPushButton" name="MoveZMinus">
                <property name="text">
                 <string>&lt;</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0" colspan="6">
               <widget class="QTextBrowser" name="DialogWindow"/>
              </item>
              <item row="0" column="5">
               <widget class="QPushButton" name="MoveXPlus">
                <property name="text">
                 <string>&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_config">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="5">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Voltage Source 1</string>
                </property>
               </widget>
              </item>
              <item row="10" column="1">
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Osciloscope</string>
                </property>
               </widget>
              </item>
              <item row="10" column="6">
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="7" column="1">
               <widget class="QCheckBox" name="Channel3">
                <property name="text">
                 <string>Channel 3</string>
                </property>
               </widget>
              </item>
              <item row="10" column="2">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="6" column="1">
               <widget class="QCheckBox" name="Channel2">
                <property name="text">
                 <string>Channel 2</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Translation Stage</string>
                </property>
               </widget>
              </item>
              <item row="9" column="1">
               <widget class="QCheckBox" name="Channel4">
                <property name="text">
                 <string>Channel 4</string>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>Connection Protocol</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEdit"/>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>Connection Protocol</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>Adress</string>
                </property>
               </widget>
              </item>
              <item row="0" column="6">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Voltage Sorce 2</string>
                </property>
               </widget>
              </item>
              <item row="1" column="5">
               <widget class="QLabel" name="label_24">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="label_19">
                <property name="text">
                 <string>Adress</string>
                </property>
               </widget>
              </item>
              <item row="10" column="5">
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="5" column="1">
               <widget class="QCheckBox" name="ChannelOne">
                <property name="text">
                 <string>Channel 1</string>
                </property>
               </widget>
              </item>
              <item row="3" column="6">
               <widget class="QLabel" name="label_27">
                <property name="text">
                 <string>Connection Protocol</string>
                </property>
               </widget>
              </item>
              <item row="2" column="6">
               <widget class="QLineEdit" name="lineEdit_8"/>
              </item>
              <item row="2" column="2">
               <widget class="QLineEdit" name="lineEdit_3"/>
              </item>
              <item row="3" column="5">
               <widget class="QLabel" name="label_26">
                <property name="text">
                 <string>Connection Protocol</string>
                </property>
               </widget>
              </item>
              <item row="2" column="5">
               <widget class="QLineEdit" name="lineEdit_7"/>
              </item>
              <item row="1" column="6">
               <widget class="QLabel" name="label_25">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item row="3" column="3" colspan="2">
               <widget class="QLabel" name="label_23">
                <property name="text">
                 <string>Connection Protocol</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3" colspan="2">
               <widget class="QLineEdit" name="lineEdit_5"/>
              </item>
              <item row="10" column="3" colspan="2">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="3" colspan="2">
               <widget class="QLabel" name="label_22">
                <property name="text">
                 <string>Adress</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3" colspan="2">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Laser</string>
                </property>
               </widget>
              </item>
              <item row="7" column="3">
               <widget class="QPushButton" name="pushButton_4">
                <property name="text">
                 <string>On</string>
                </property>
               </widget>
              </item>
              <item row="7" column="4">
               <widget class="QPushButton" name="pushButton_5">
                <property name="text">
                 <string>Off</string>
                </property>
               </widget>
              </item>
              <item row="5" column="3" colspan="2">
               <widget class="QLabel" name="label_28">
                <property name="text">
                 <string>Frequency [Hz]</string>
                </property>
               </widget>
              </item>
              <item row="6" column="3" colspan="2">
               <widget class="QSpinBox" name="spinBox">
                <property name="maximum">
                 <number>500000</number>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QComboBox" name="comboBox"/>
              </item>
              <item row="4" column="2">
               <widget class="QComboBox" name="comboBox_2"/>
              </item>
              <item row="4" column="3" colspan="2">
               <widget class="QComboBox" name="comboBox_3"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_scan">
           <attribute name="title">
            <string>Page</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>17</x>
              <y>27</y>
              <width>130</width>
              <height>358</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>318</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <spacer name="horizontalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="start_daq">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>80</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Start DAQ</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QPushButton" name="save_config">
            <property name="text">
             <string>Save Config</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="load_config">
            <property name="text">
             <string>Load Config</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="initialize">
            <property name="text">
             <string>Initialize</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupStatus">
        <property name="minimumSize">
         <size>
          <width>301</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>301</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Instruments Status</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>XSpeed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_7</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YSpeed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_8</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ZSpeed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_9</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
